doc/doxygen/src/driver-guide.md

```diff
@@ -159,30 +166,38 @@ CFLAGS="-DTMPABC_PARAM_ADDR=0x23" make all
 Second, we can override selected parameters from the board configuration
 (`board.h`):
 
-    /**
-    * @brief   TMPABC sensor configuration
-    * @{
-    */
-    #define TMPABC_PARAM_I2C        (I2C_DEV(1))
-    #define TMPABC_PARAM_ADDR       (0x17)
-    /** @} */
+@code.{c}
```

Comment:
```suggestion
@code{.c}
```

Reply:
I guess this needs to be just `@code` then.

Reply:
Yup